dts/bindings/power-domain/intel,power-domain.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Power domains for intel ADSP
+
+compatible: "intel,power-domain"
```

Comment:
This is not generic for all platforms, it should be adsp specific.

Reply:
Wrong copy. We changed it a while back, corrected